cli/bpmetadata/tfconfig.go

```diff
@@ -256,6 +269,34 @@ func getBlueprintInterfaces(configPath string) (*BlueprintInterface, error) {
 	}, nil
 }
 
+func getBlueprintVariableOrders(configPath string) map[string]int {
+	p := hclparse.NewParser()
+	variableFile, hclDiags := p.ParseHCLFile(filepath.Join(configPath, "variables.tf"))
+	if hclDiags.HasErrors() {
+		Log.Error("Failed to parse HCL: ", "diags: ", hclDiags)
+	}
+	variableContent, _, hclDiags := variableFile.Body.PartialContent(variableSchema)
+	if hclDiags.HasErrors() {
+		Log.Error("Failed to parse variable content: %v", hclDiags)
+	}
+	variableOrderKeys := make(map[string]int)
+	for i, block := range variableContent.Blocks {
+		// We only care about variable blocks.
+		if block.Type != "variable" {
+			continue
+		}
+		// We expect a single label which is the variable name.
+		if len(block.Labels) != 1 {
+			Log.Info("Variable block has no name: %v", block)
+			continue
+		}
+		variableOrderKeys[block.Labels[0]] = i
+
+	}
+	Log.Info("Found variables in order: ", "variableOrderKeys: ", variableOrderKeys)
```

Comment:
Let's remove this, I had it just for debugging in the internal poc

Reply:
Done